tst/Carnival/CarnivalTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\Carnival;
+
+use Mockery;
+use Mockery\Exception\InvalidCountException;
+use PHPUnit\Framework\TestCase;
+
+class CarnivalTest extends TestCase {
```

Comment:
abstract class?

Reply:
Also, this isn't a "Test", technically. It's an extension of the TestCase class. So perhaps this should be `CarnivalTestCase`. Thoughts?

Reply:
Yeah sure, I'll make it an abstract class and change its name.